styles/style.css

```diff
@@ -0,0 +1,50 @@
+* {
+    margin: 0;
+}
+
+.section {
+    max-height: 569px;
```

Comment:
If you really need `height` here(actually you don't need), at least use `min-height` instead of `max-height`